index.css

```diff
@@ -0,0 +1,242 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300,400,500,600&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: c, sans-serif;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  max-width: 1200px;
+  height: 100vh;
+  margin: auto;
+  padding: 0;
+  background-image: url(./book-bg.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  border: 4px solid black;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 15px;
+  margin-top: 5px;
+}
+
+.logo img {
+  width: 230px;
+}
+
+ul.menu {
+  color: #000;
+  display: flex;
+}
+
+ul.menu li a {
+  font-size: 1.2rem;
+  color: rgb(0, 135, 177);
+}
+
+ul.menu li a:hover {
+  color: #000;
+  border: 1px solid orangered;
+  border-radius: 6px;
+  padding: 0.5rem;
+}
+
+span {
+  padding: 0 13px;
+}
+
+.current-date {
+  text-align: right;
+
+  /* color: rgb(247,131,50); */
```

Comment:
- [ ] for good practice, Kindly, remove this comment line 👍

Reply:
They were all removed but 😄 Thanks